eng/tools/mcp/azure-sdk-java-mcp/ci.yml

```diff
@@ -0,0 +1,115 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+trigger:
+  branches:
+    include:
+      - main
+      - feature/*
+      - release/*
+      - hotfix/*
+  paths:
+    include:
+      - tools/mcp/azure-sdk-java-mcp
+
+pr:
+  branches:
+    include:
+      - main
+      - feature/*
+      - release/*
+      - hotfix/*
+  paths:
+    include:
+      - tools/mcp/azure-sdk-java-mcp
+
+extends:
+  template: /eng/pipelines/templates/stages/1es-redirect.yml
+  parameters:
+    stages:
+      - stage: 'Build'
+        variables:
+          - template: /eng/pipelines/templates/variables/image.yml
+          - name: NodeVersion
+            value: '18.x'
+          - name: ArtifactName
+            value: 'azure-sdk-java-mcp'
+          - name: TypeScriptGeneratorDirectory
```

Comment:
Rename this variable to make it more readable as azure-sdk-java-mcp project path